decks/food.mdx

```diff
@@ -9,371 +9,526 @@ export const theme = customTheme;
 
 ---
 
-<L.Row>
+- Fill in user says as below
 
-## Oops #1
+import chatbot22 from '../assets/2_2.png';
 
-### Delaying rendering is only safe when no effects are pending on a subtree.
+<img src={chatbot22} />
 
-</L.Row>
 
 ---
 
-import HooksAPI from '../assets/hooks-api.svg';
-
-<L.Row>
-
-## The Hooks API.
+- Scroll down to the “Responses” tab
+- Delete the welcome messages
+- Fill in "Responses" as below
+- Click "Save"
 
-### This restriction is reflected in the hooks API with encapsulation of effects.
+import chatbot23 from '../assets/2_3.png';
 
-</L.Row>
+<img src={chatbot23} />
 
-<HooksAPI />
 
 ---
 
-import HooksAPITiming from '../assets/hooks-api-timing.svg';
-
 <L.Row>
 
-## Hooks Timings.
+## 3. Creating a Custom Entity
 
-### With each hook we observe a part of the lifecycle and can react.
+### Entities are what we extract from user’s input to process further
 
 </L.Row>
 
-<HooksAPITiming />
+<Background />
 
 ---
 
-import ComponentLifecycle from '../assets/component-lifecycle.svg';
+- Click on the “Entities” tab on left-sidebar and click “Create Entity”
+- Name our Entity “content”
+- Fill in the following details
 
-<L.Row>
+import chatbot31 from '../assets/3_Entity.png';
 
-## The Lifecycle.
+<img src={chatbot31} />
 
-### The lifecycle we observe has three phases repeating.
-
-</L.Row>
-
-<ComponentLifecycle />
 
 ---
 
-import ObservingLifecycleEvents from '../assets/observing-lifecycle-events.svg';
-
 <L.Row>
 
-## Observing the Lifecycle.
-
-### Overlapping the lifecycle with hooks, we see a phase that we can't observe...
+## 4. Attaching our new Entity to the Intent
+### Create a new Intent called “say-content”
 
 </L.Row>
 
-<ObservingLifecycleEvents />
+<Background />
 
 ---
 
-import InterruptedLifecycle from '../assets/interrupted-lifecycle.svg';
+- Add the phrase “Let’s hear a joke” in the “User Says” section
 
-<L.Row>
+import chatbot32 from '../assets/3_2.png';
 
-## Interrupting the Lifecycle.
+<img src={chatbot32} />
 
-### Unfortunately this phase becomes very relevant in Strict and Concurrent Mode.
 
-</L.Row>
+---
+- Let’s add a few more inputs
+
+import chatbot33 from '../assets/3_3.png';
+
+<img src={chatbot33} />
 
-<InterruptedLifecycle />
 
 ---
+- Let’s add a placeholder text response to see it work
+- scroll to the bottom section “Response”, and fill it like so
 
-<L.Row>
+import chatbot34 from '../assets/3_4.png';
 
-## Oops #2
+<img src={chatbot34} />
 
-### It's not possible to have a subscription or effect track the component lifecycle.
 
-</L.Row>
+---
+- Click on “Try It Now” to test your work
+
+import chatbot35 from '../assets/3_5.png';
+
+<img src={chatbot35} />
+
 
 ---
 
-<L.Row>
+<L.Column>
 
-## Workaround
+# 5. A Webhook to process requests
+- Write a JS app that fulfills the request by querying the Reddit’s website and returning the appropriate content.
 
-### Resort to useSubscription
+</L.Column>
 
-</L.Row>
+---
+# code
+---
+
+<CodeSurfer>
+
+​```js
+'use strict';
```

Comment:
this slide is not very helpful. the code should be split up into sections and each section should be briefly explained. And the whole code should be available in a GitHub Gist